pages/dao/[symbol]/proposal/components/instructions/TokenTransferBetweenInternalGovernanceAccounts.tsx

```diff
@@ -0,0 +1,208 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import React, { useContext, useEffect, useState } from 'react'
+import * as yup from 'yup'
+import { isFormValid } from '@utils/formValidation'
+import { Token, TOKEN_PROGRAM_ID } from '@solana/spl-token'
+import BigNumber from 'bignumber.js'
+import {
+  UiInstruction,
+  TokenTransferBetweenInternalGovernanceAccountsForm,
+} from '@utils/uiTypes/proposalCreationTypes'
+import { NewProposalContext } from '../../new'
+import useWalletStore from 'stores/useWalletStore'
+import {
+  ProgramAccount,
+  serializeInstructionToBase64,
+  Governance,
+} from '@solana/spl-governance'
+import GovernedAccountSelect from '../GovernedAccountSelect'
+import useGovernedMultiTypeAccounts from '@hooks/useGovernedMultiTypeAccounts'
+import Input from '@components/inputs/Input'
+import useGovernanceUnderlyingTokenAccounts from '@hooks/useGovernanceUnderlyingTokenAccounts'
+import TokenAccountSelect from './TokenAccountSelect'
+
+const TokenTransferBetweenInternalGovernanceAccounts = ({
+  index,
+  governance,
+}: {
+  index: number
+  governance: ProgramAccount<Governance> | null
+}) => {
+  const connection = useWalletStore((s) => s.connection)
+  const wallet = useWalletStore((s) => s.current)
+  const { governedMultiTypeAccounts } = useGovernedMultiTypeAccounts()
+
+  // Hardcoded gate used to be clear about what cluster is supported for now
+  if (connection.cluster !== 'mainnet') {
+    return <>This instruction does not support {connection.cluster}</>
```

Comment:
is there a reason for this instruction not to be supported on devnet / testnet ?

Reply:
Actually no, I've re-enabled it :)